policyengine_us/tests/policy/baseline/gov/states/or/tax/income/or_liheap_region.yaml

```diff
@@ -0,0 +1,8 @@
+- name: Oregon LIHEAP region 1 
+  period: 2021
+  input: 
+    spm_unit_size: 1 
+    county: CURRY_COUNTY_OR
+    state_code: OR
+  output: 
+    region: REGION 1
```

Comment:
```suggestion
    county_str: CURRY_COUNTY_OR
    state_code: OR
  output: 
    or_liheap_in_region_one: REGION 1
```

Reply:
Make sure the output mirrors the class and the inputs are only variables used in the specific formula